be/src/olap/rowset/beta_rowset_writer.cpp

```diff
@@ -340,16 +338,19 @@ Status BetaRowsetWriter::_do_compact_segments(SegCompactionCandidatesSharedPtr s
         for (const auto& entry : std::filesystem::directory_iterator(_context.rowset_dir)) {
             fmt::format_to(vlog_buffer, "[{}]", string(entry.path()));
         }
-        VLOG_DEBUG << "_segcompacted_point:" << _segcompacted_point
+        VLOG_DEBUG << "tablet_id:" << _context.tablet_id << " rowset_id:" << _context.rowset_id
+                   << "_segcompacted_point:" << _segcompacted_point
                    << " _num_segment:" << _num_segment << " _num_segcompacted:" << _num_segcompacted
                    << " list directory:" << fmt::to_string(vlog_buffer);
     }
 
     _segcompacted_point += (end - begin + 1);
 
     uint64_t elapsed = GetCurrentTimeMicros() - begin_time;
-    LOG(INFO) << "BetaRowsetWriter:" << this << " segcompaction completed. elapsed time:" << elapsed
-              << "us. _segcompacted_point update:" << _segcompacted_point;
+    LOG(INFO) << "segcompaction completed. tablet_id:" << _context.tablet_id
```

Comment:
Maybe `fmt::format` is better.

Reply:
Better performance, you can refer to: https://stackoverflow.com/questions/58157033/why-is-fmt-slower-than-stdstringstream

Reply:
I do not think we should use fmt in log... It is very ugly. If it is required, I think it is better to add another logger, like LOG_INFO(fmt, args....);

Reply:
Thanks for explanation. Now I am a little confused at when I should use which ... I will benchmark them in my spare time.